testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -167,3 +167,11 @@ Feature: Smoke tests for <client>
     And I visit "Prometheus apache exporter" endpoint of this "<client>"
     And I wait until "postgres" exporter service is active on "<client>"
     And I visit "Prometheus postgres exporter" endpoint of this "<client>"
+
+  Scenario: Test <client> on Grafana
+    When I visit the grafana dashboards of this "monitoring_server"
+    And I wait until I do not see "Loading Grafana" text
+    When I follow "Client Systems"
```

Comment:
Woudn't `And` work better here?

Reply:
I didn't use it because I saw this as the "actual start" of the actions, if it makes sense :thinking: but I'll change it if others agree with you, I don't have really strong feelings either way, I just don't like too many `And`s